webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -14438,6 +14460,10 @@ var userOverride = {
                             $file_remove.trigger('click');
                         }
                     });
+                    var ATTACHMENT_UPLOAD_URL = '/rest/ws/upload/image';
```

Comment:
Can we keep the constant definition at the same place as before? And add just the required parameter here without editing constant?